src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -26,30 +26,36 @@
  * Calculate genotype posterior probabilities given family and/or known population genotypes
  *
  * <p>
- * This tool calculates the posterior genotype probability for each sample genotype in a VCF of input variant calls,
- * based on the genotype likelihoods from the samples themselves and, optionally, from input VCFs describing allele
- * frequencies in related populations. The input variants must possess genotype likelihoods generated by
- * HaplotypeCaller, UnifiedGenotyper or another source that provides <b>unbiased</b> genotype likelihoods.</p>
+ * The tool calculates the posterior genotype probability for each sample genotype in a given VCF format callset.
+ * The input variants must present genotype likelihoods generated by HaplotypeCaller, UnifiedGenotyper or other
+ * source that provides unbiased genotype likelihoods.</p>
```

Comment:
Now that I think about it some more, maybe Chris meant that you should make sure priors haven't already been applied?  Not that it matters.  It's a true statement so we'll leave it in.

Reply:
Ok, thanks for sharing what it could mean.